### PR TITLE
Integration tests reopen

### PR DESCRIPTION
**Pull request by pf15ese**
- added test db
- added controller integration tests using the db
- tried to clean up test folder
- Integration tests are now called in the github workflow

Since only one test db is provided we need to run the integration tests sequentially with --runInBand.

You may need to drop volumes to run the test correctly locally:

docker-compose down --volumes

You may also need to updated your .env based on the new .env.example

### CODE REVIEW

**Comment by imscaradh** [comment]
@pf15ese please resolve the conflicts

File: docker-compose.yaml [hunk 1]
```diff
@@ -37,6 +37,7 @@ services:
       - ${NODE_PORT}:8000
     volumes:
       - /etc/localtime:/etc/localtime:ro
+      - .:/app
```
Comment (imscaradh) [inline docker-compose.yaml]: is there a reason this is being mounted here?

File: resources/init-postgres/create-db.sh [hunk 2]
```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
```
Comment (imscaradh) [inline resources/init-postgres/create-db.sh]: Can you please either bootstrap all data from init.sql or restore the init.sql script?

File: src/server.ts [hunk 3]
```diff
@@ -3,17 +3,13 @@ import {InfluxDB} from 'influx';
...
 import MQTTTopicClient from './mqtt/client';
 import {influxConn, pgConn} from './persistence/database';
 import {routes} from './routes';
 import CoffeeDetector from './service/coffeeDetector';
-import DataRecorder from './service/recorder/dataRecorder';
-import {InfluxDataRecorder} from './service/recorder/influxDataRecorder';
-import { gravityTransformerTagged } from './service/thingy';
 
-async function bootstrap(samples: boolean) {
+async function bootstrap() {
```
Comment (imscaradh) [inline src/server.ts]: Why did you remove this part? It can be helpful to work with sample data even no tests are executed (backend-frontend development)

File: docker-compose.yaml (hunk 1 above)
Comment (pf15ese) [reply docker-compose.yaml]: No, is gone
